src/middleware/access.ts

```diff
@@ -2,41 +2,41 @@ import { Request, Response, NextFunction } from "express";
 //import { logger } from "../utils/logger.js";
 
 const middleware = (req: Request, res: Response, next: NextFunction): void => {
-    const userAgent = req.get("User-Agent");
+    // const userAgent = req.get("User-Agent");
```

Comment:
and this is why you don't do `git add .` all the time.

This is good for local development but on production we don't want this

Reply:
this slipped my mind srry, i was testnig on my wiiu but over http, these headers dont get sent. only x-nintendo-release-version